pkgs/development/libraries/nanopb/default.nix

```diff
@@ -27,7 +28,7 @@ stdenv.mkDerivation rec {
     "-DBUILD_SHARED_LIBS=ON" # generate $out/lib/libprotobuf-nanopb.so{.0,}
     "-DBUILD_STATIC_LIBS=ON" # generate $out/lib/libprotobuf-nanopb.a
     "-Dnanopb_PROTOC_PATH=${buildPackages.protobuf}/bin/protoc"
-  ];
+  ] ++ (if mallocBuild then [ "-DCMAKE_C_FLAGS=-DPB_ENABLE_MALLOC 1" ] else []);
```

Comment:
```suggestion
  ] ++ lib.poptionals mallocBuild [ "-DCMAKE_C_FLAGS=-DPB_ENABLE_MALLOC 1" ];
```

Reply:
`lib.poptionals` says attribute `poptionals` is missing. Can I use `lib.lists.optional` (since there's only one element) instead?

Reply:
I meant lib.optionals.